### DISCUSSION TITLE
Stack or Queue (nginx)

### DISCUSSION THREAD

**Discussion by JoeMaputo** [community/community#18646] [Other Feature Feedback, Questions, & Ideas]
NGINX is used as a web server.
I wrote a small script that stores the request number in a database table (you can write to a file).
The request number is passed in GET parameters. (for example, ?number=1)
Requests were generated in the amount of 1000 pieces using a Java script in the browser console without a time delay between requests.
It is expected that in the database (in the file) the numbers will be in order.
In fact (for an 8-core processor), the numbers go 8 at a time in reverse order. It seems that the request that came last is processed first.
Is it possible to somehow configure this so that requests are processed by a queue and not a stack?
#nginx

**Comment by MathiasReker** [comment]
You can sort the database asc/desc by time before running the stack :)

**Reply by JoeMaputo** [reply]
Thanks for answer.
It's not about getting the right order with a SQL query.
It is necessary that requests are processed by the script one by one.
